changelogs/fragments/58381_postgresql_ext-add-version-parameter.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- postgresql_ext - add version parameter to support creation / update extensions of specific versions.
```

Can you add the link to the PR here please?
```suggestion
- postgresql_ext - add version parameter to support creation / update extensions of specific versions (https://github.com/ansible/ansible/pull/58381)
```

---

I thought that it needs when issues are presented. Ok, nps. I committed your suggestion, thanks :)